packages/tasks/src/model-libraries-snippets.ts

```diff
@@ -1032,4 +1032,12 @@ export const threedtopia_xl = (model: ModelData): string[] => [
 model = threedtopia_xl.from_pretrained("${model.id}")
 model.generate(cond="path/to/image.png")`,
 ];
+
+export const hezar = (model: ModelData): string[] => [
+	`# !pip install hezar
+from hezar import Model
+
+model = Model.load("${model.id}")
+outputs = model.predict("a valid input based on modality (text, path, array, etc.)")`,
```

Comment:
To make the example even better, you can check the model task using `model.pipeline_tag`. This way, you should know which modality to use.

This can be done in a later PR if you prefer. In the meantime, I would remove the `model.predict` line if that's the case and keep only the `Model.load` one.

Reply:
Done🤗. I just kept the `Model.load` and removed `pip install` instruction.